libbeat/docs/shared-ssl-config.asciidoc

```diff
@@ -87,25 +87,154 @@ Example module with SSL enabled:
 ----
 endif::[]
 
-[float]
-=== Configuration options
+[discrete]
+[[ssl-common-config]]
+=== Common configuration options
 
-You can specify the following options in the `ssl` section of the +{beatname_lc}.yml+ config file:
+In the +{beatname_lc}.yml+ config file, you can specify the following common options in the
+`ssl` section of each output that supports SSL.
```

Comment:
not just output, but almost anything that is network facing. e.g. inputs, metricsets, even potentially auto-discovery or modules that need to talk to remote services.

Reply:
To avoid any confusion., I made this more generic.